src/components/container/DocumentLibraryPreview/DocumentLibraryPreview.css

```diff
@@ -0,0 +1,15 @@
+.mdhui-document-library-preview .mdhui-document-library-preview-title-icon {
+    width: 15px;
+    height: 15px;
+    color: var(--mdhui-text-color-0);
+    margin-right: 8px;
+    margin-bottom: 2px;
+}
+
+.mdhui-document-library-preview .mdhui-document-library-preview-documents {
+    margin-top: 8px;
+}
+
+.mdhui-document-library-preview .mdhui-document-library-preview-document-name {
+    color: var(--mdhui-text-color-3);
```

Comment:
This text color doesn't pass accessbility standards in either dark mode or light mode: https://webaim.org/resources/contrastchecker/?fcolor=59595D&bcolor=2C2C2D / https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=2C2C2D

Reply:
I've switched to `--mdhui-text-color-2`.  That scores much better on both.

Light: https://webaim.org/resources/contrastchecker/?fcolor=555555&bcolor=FFFFFF
Dark: https://webaim.org/resources/contrastchecker/?fcolor=9E9EA5&bcolor=2C2C2D